docs/aqua-book/changelog.md

```diff
@@ -7,6 +7,20 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+## [0.13.0](https://github.com/fluencelabs/aqua/releases/tag/aqua-v0.13.0) - November 22, 2023
+
+* BREAKING CHANGE: **compiler:** Make `nil` of option of bottom type ([LNG-279](https://linear.app/fluence/issue/LNG-279)) ([#968](https://github.com/fluencelabs/aqua/issues/968))
```

Comment:
What does it mean?

Reply:
It means that literal value `nil` has type `?bottom`. If you have any suggestions on how to rephrase this, feel free to make a suggestion.

Reply:
> Breaking change: `nil` type changed from ???WHAT??? to `?⟂`, see THIS PAGE to learn about how to update your code